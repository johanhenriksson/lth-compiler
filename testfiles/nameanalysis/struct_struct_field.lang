struct outer {
    test t;
}

struct test {
    int hello;
}

void main() {
    outer o;
    o.t.hello = 5; // ok
    o.t.undef = 6; // error
}
