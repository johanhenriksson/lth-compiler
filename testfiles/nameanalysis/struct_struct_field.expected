Error at line 12,5: no such field 'undef' on struct t of type test
